### PR TITLE
Remove version from tiup playground and add two examples

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/8455

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-playground.md [hunk 1]
```diff
@@ -63,25 +63,26 @@ Flags:
...
 ## 使用示例
 
-### 使用每日构建版启动一个 TiDB 集群
+### 启动一个指定版本的 TiDB 集群
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-tiup playground nightly
+tiup playground version
```
Comment (qiancai) [inline tiup/tiup-playground.md:71]:
```suggestion
tiup playground ${version}
```

File: tiup/tiup-playground.md [hunk 2]
```diff
@@ -63,25 +63,26 @@ Flags:
...
 ```
 
-nightly 就是这个集群的版本号，这里指定为每日构建版本。
+将 version 替换为实际需要的版本号。
```
Comment (qiancai) [inline tiup/tiup-playground.md:74]:
```suggestion
将 ${version} 替换为所需的版本号。
```

File: tiup/tiup-playground.md [hunk 3]
```diff
@@ -63,25 +63,26 @@ Flags:
...
 
-### 使用每日构建版启动一个 TiDB 集群
+### 启动一个指定版本的 TiDB 集群
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-tiup playground nightly
+tiup playground version
 ```
```
Comment (qqqdan) [inline tiup/tiup-playground.md:72]: tiup playground [version]

File: tiup/tiup-playground.md [hunk 4]
```diff
@@ -63,25 +63,34 @@ Flags:
...
-### 启动一个带监控的集群
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup playground ${version}
+```
+
+将 ${version} 替换为所需的版本号。
+
+### 使用每日构建版启动一个 TiDB 集群
```
Comment (TomShawn) [inline tiup/tiup-playground.md:84]: 要在 v5.4 或其他 release 文档中说明如何使用 nightly 版本吗？

File: tiup/tiup-playground.md [hunk 5]
```diff
@@ -110,7 +119,7 @@ tiup playground --db.binpath /xx/tidb-server
 {{< copyable "shell-regular" >}}
 
 ```shell
-tiup playground v5.4.0 --db 3 --pd 3 --kv 3
+tiup playground --db 3 --pd 3 --kv 3
```
Comment (TomShawn) [inline tiup/tiup-playground.md:122]: 去掉版本默认使用最新的 nightly 吗？

**Review by qqqdan — APPROVED** [review]

File: tiup/tiup-playground.md (hunk 4 above)
Comment (shichun-0415) [reply tiup/tiup-playground.md:84]: 我理解是需要的，这个 PR 会 CP 到其他分支的。

File: tiup/tiup-playground.md (hunk 5 above)
Comment (shichun-0415) [reply tiup/tiup-playground.md:122]: 因为前面已经描述过不加版本号会直接用最新版本，前面也加了示例告诉用户怎么指定版本，所以我觉得这个地方这么描述是可以的，而且这个地方的命令侧重点在启动多个组件实例，所以突然强调版本号也会不合适。

File: tiup/tiup-playground.md (hunk 4 above)
Comment (TomShawn) [reply tiup/tiup-playground.md:84]:
```suggestion
### 启动一个每日构建版的 TiDB 集群
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

**Review by qiancai — APPROVED** [review]
LGTM

File: tiup/tiup-playground.md [hunk 6]
```diff
@@ -63,25 +63,34 @@ Flags:
...
+### 启动一个指定版本的 TiDB 集群
+
+{{< copyable "shell-regular" >}}
 
-### 启动一个带监控的集群
+```shell
+tiup playground ${version}
+```
+
+将 ${version} 替换为所需的版本号。
```
Comment (qiancai) [inline tiup/tiup-playground.md:82]:
```suggestion
将 `${version}` 替换为所需的版本号。
```

File: tiup/tiup-playground.md [hunk 7]
```diff
@@ -63,25 +63,34 @@ Flags:
...
 {{< copyable "shell-regular" >}}
 
 ```shell
-tiup playground nightly 
+tiup playground nightly
 ```
 
-该命令会在 9090 端口启动 Prometheus 用于展示集群内部的时序数据。
+nightly 就是这个集群的版本号，这里指定为每日构建版本。
+
```
Comment (shichun-0415) [inline tiup/tiup-playground.md:93]:
```suggestion
```

**Pull request merged**